examples/discord_bot.py

```diff
@@ -52,18 +52,47 @@ async def player_heroes(ctx, player_tag):
 async def clan_info(ctx, clan_tag):
     clan = await coc_client.get_clan(clan_tag)
 
+    for a in clan.members:
+        if a.role == a.role.leader:
+            leader = a
+    if clan.public_war_log is False:
+        log = 'Private'
+    else:
+        log = 'Public'
     e = discord.Embed(colour=discord.Colour.green())
     e.set_thumbnail(url=clan.badge.url)
-    e.add_field(name=clan.name, value=clan.tag)
-    e.add_field(name="Description", value=clan.description)
-    e.add_field(name="Members", value=", ".join([member.name for member in clan.members]))
-    e.add_field(
-        name="Clan Record", value="{}-{}-{}".format(clan.war_wins, clan.war_losses, clan.war_ties),
-    )
-
+    e.add_field(name='Clan Name', value=f'{clan.name}({clan.tag})\n[Open in game]({clan.share_link})', inline=False)
+    e.add_field(name='Clan Level', value=clan.level, inline=False)
+    e.add_field(name="Description", value=clan.description, inline=False)
+    e.add_field(name="Leader", value=f'{leader}', inline=False)
+    e.add_field(name='Clan Type', value=clan.type, inline=False)
+    e.add_field(name='Location', value=clan.location, inline=False)
+    e.add_field(name='Total Clan Trophies', value=clan.points, inline=False)
+    e.add_field(name='Total Clan Versus Trophies', value=clan.versus_points, inline=False)
+    e.add_field(name='WarLog Type', value=log, inline=False)
+    e.add_field(name= 'Required Trophies', value=clan.required_trophies, inline=False)
+    e.add_field(name='War Win Streak', value = clan.war_win_streak, inline=False)
+    e.add_field(name='War Frequency', value= clan.war_frequency, inline=False)
+    e.add_field(name='Clan War League Rank', value= clan.war_league, inline=False)
+    e.add_field(name='Clan Labels', value ='\n'.join(label.name for label in clan.labels), inline=False)
+    e.add_field(name='Member Count', value = f'{clan.member_count}/50', inline=False)
+    e.add_field(name="Clan Record", value=f"Won - {clan.war_wins}\nLost - {clan.war_losses}\n Draw - {clan.war_ties}", inline=False)
     await ctx.send(embed=e)
 
 
+@bot.command()
+async def clan_member(ctx, clan_tag):
+    clan = await coc_client.get_clan(clan_tag)
+    member = ''
+    for i, a in enumerate(clan.members, start=1):
+        member += f'``{i}.`` {a.name}\n'
+    embed = discord.Embed(colour=discord.Colour.red(), title= f'Members of {clan.name}',description= f'{member}')
```

Comment:
Same as above a bit, `member` is a string, so wrapping it inside a f-string does nothing :) also, title and description have extra spaces after the =